docs/en/install/install-dce.md

```diff
@@ -0,0 +1,151 @@
+# Install DCE 5.0
+
+This page describes the installation procedure of DCE 5.0.
+
+!!! note
+
+    Click to [watch installation video](../videos/install.md).
+
+## Preparation
+
+- Prepare a k8s cluster. See [how to deploy a k8s cluster](install-k8s.md)
+
+    !!! note
+
+        - Available cluster resources: CPU > 10 cores, memory > 12 GB, remained storage space > 100 GB (currently, DCE 5.0 provides multiple replicas by default. If you choose a single replica, the resource consumption can be reduced to 2 cores and 4 GB)
+        - Kubernetes version: The latest stable version is recommended. It supports v1.21 to v1.24 currrently.
+        - Supported CRIs: Docker and containerd
+        - Storage: Prepare StorageClass and set it to the default. For details see [how to deploy a k8s cluster](install-k8s.md)
+        - Only support X86_64 currently
+        - Ensure your cluster has installed with CoreDNS
+    
+- Install [dependencies](install-tools.md)
+
+    !!! note
+
+        If you have installed all required dependencies in your cluster, ensure all your dependencies are correctly installed:
+
+        - helm ≥ 3.9.4
+        - skopeo ≥ 1.9.2
+        - kubectl ≥ 1.22.0
+        - yq ≥ 4.27.5
+
+## Online installation (recommended)
+
+1. On the k8s control plane (master node), download the dce5-installer binary package.
+
+    ```shell
+    # Assume VERSION is v0.3.12
+    export VERSION=v0.3.12
+    curl -Lo ./dce5-installer https://qiniu-download-public.daocloud.io/DaoCloud_Enterprise/dce5/dce5-installer-${VERSION}
+    ```
+
+    Add the executable permission to `dce5-installer`:
+
+    ```bash
+    chmod +x dce5-installer
+    ```
+
+2. Edit your `clusterConfig.yaml`
+
+    - For a private cloud (virtual and physical machines), enable your load balancer (metallb) to avoid the NodePort instability caused by IP variations. Carefully plan your network and IP addresses. Here is an example for your reference:
+
+        ```yaml
+        apiVersion: provision.daocloud.io/v1alpha1
+        kind: ClusterConfig
+        spec:
+        LoadBalancer: metallb
+        istioGatewayVip: 10.6.229.10/32     # This is VIP for Istio gateway and is also the URL via which you can use DCE 5.0 on your web browser.
+        insightVip: 10.6.229.11/32          # This is the VIP used by the Insight-Server of the Global cluster to collect logs, metrics, and traces of all sub-clusters.
+        ```
+
+    - For a public cloud, DCE integrates a Cloud Controller Manager to provide the load balancing capability for kubernetes. Here is an example for your reference:
+
+        ``` yaml
+        apiVersion: provision.daocloud.io/v1alpha1
+        kind: ClusterConfig
+        spec:
+        LoadBalancer: cloudLB
+        ```
+
+    - If you use NodePort to expose your console (for proof of concept (PoC)), you can skip the clusterConfig file and go to the next step.
+
+3. Install DCE 5.0.
```

Comment:
Do we need to change all "DCE 5.0" into "DCE 5.0 Community Package", since what installed here is just the community package.